docs/public-networks/get-started/connect/sync-node.md

```diff
@@ -4,39 +4,132 @@ description: Full and archive node types
 
 # Sync Besu
 
-Besu supports two node types, commonly referred to as _full nodes_ and _archive nodes_.
+Besu supports two node types, commonly referred to as [full nodes](#run-a-full-node) and
+[archive nodes](#run-an-archive-node).
 
-Full nodes have the current state of the blockchain so cannot serve the network with all data
-requests (for example, the balance of an account at an old block). Full nodes can guarantee the
-latest state for the blockchain (and some older states, but not all). You can check current
-balances, sign and send transactions, and look at current dapp data.
+Full nodes have the current state of the blockchain.
+They can't serve the network with all data requests (for example, the balance of an account at an
+old block).
+Full nodes can guarantee the latest state for the blockchain (and some older states, but not all).
+You can check current balances, sign and send transactions, and look at current dapp data.
 
-Archive nodes have all of this and they also store the intermediary state of every account and
-contract for every block since the genesis block. An archive node can do everything a full node
-does, and it can access historical state data.
+Archive nodes also store the intermediary state of every account and contract for every block since
+the genesis block.
+Archive nodes can do everything full nodes do, and they can access historical state data.
+Archive nodes require more disk space than full nodes.
 
-For Besu on Mainnet, archive nodes [require more disk space](../../concepts/data-storage-formats.md#storage-requirements)
-than full nodes.
+## Sync times
+
+To sync with a public network, Besu runs two processes in parallel: the world state sync and the
+blockchain download.
+
+The following table shows the average world state sync time for each sync mode.
```

Comment:
Call out that this table is for mainnet?

Maybe could have another table for goerli/sepolia? I think most stakers test on goerli so they can try out setting up a validator. 

Or maybe could just say "testnets will take significantly less time and space to sync"?